boefjes/boefjes/local_repository.py

```diff
@@ -107,66 +103,90 @@ def description_path(self, id_: str) -> Path | None:
         return boefjes[id_].path / "description.md"
 
     def resolve_boefjes(self) -> dict[str, BoefjeResource]:
-        if self._cached_boefjes:
-            return self._cached_boefjes
+        return _cached_resolve_boefjes(self.path)
 
-        paths_and_packages = self._find_packages_in_path_containing_files([BOEFJE_DEFINITION_FILE])
-        boefje_resources = []
+    def resolve_normalizers(self) -> dict[str, NormalizerResource]:
+        return _cached_resolve_normalizers(self.path)
 
-        for path, package in paths_and_packages:
-            try:
-                boefje_resources.append(BoefjeResource(path, package))
-            except ModuleException as exc:
-                logger.exception(exc)
 
-        self._cached_boefjes = {resource.boefje.id: resource for resource in boefje_resources}
+@cache
+def _cached_resolve_boefjes(path) -> dict[str, BoefjeResource]:
+    """Adding the hash to the arguments makes sure we refresh this. The size could hence be 1, but since this is not
```

Comment:
There isn't a hash in the arguments, only the path, was the docstring was written for a previous version?

Reply:
Indeed!